.github/workflows/scripts/build_npm.ts

```diff
@@ -0,0 +1,81 @@
+import { build, emptyDir } from "https://deno.land/x/dnt@0.22.0/mod.ts";
+import { parse } from "https://deno.land/std@0.133.0/flags/mod.ts";
+
+const entryPointDefault = "./mod.ts";
+const outDirDefault = "./.github/workflows/dist";
+
+const helpText =
+  "Deno to node transpiler, converts the ts_serialize module to a node compatible module, and writes it to .github/workflows/npm, or the directory of your choice." +
+  "\n" +
+  "\nThis must be run from the project root if using the default arguments." +
+  "\n" +
+  "\nUsage:" +
+  `\n\tdeno run -A .github/workflows/scripts/build_npm.ts -v "v0.0.0-test" [-e ${entryPointDefault}] [-o ${outDirDefault}]` +
+  "\n\tdeno run -A .github/workflows/scripts/build_npm.ts --help" +
+  "\n" +
+  "\nCommand line arguments:" +
+  "\n\t-h,  --help               Prints this help message, then exits." +
+  "\n\t-v,  --version=[version]  The version value to set in the resulting package.json. Required." +
+  `\n\t-e,  --entry-point=[path] The path to the mod.ts file to transpile into the npm package. Defaults to "${entryPointDefault}"` +
+  `\n\t-o,  --out=[path]         The path to the target directory where the built files should be placed. Defaults to "${outDirDefault}"`;
+
+function printHelpText() {
+  console.log(helpText);
+}
+
+async function main() {
+  const flags = parse(Deno.args);
+
+  if (flags["h"] || flags["help"]) {
+    printHelpText();
+    return;
+  }
+
+  const version = flags["v"] || flags["version"];
+  const entryPoint = flags["e"] || flags["entry-point"] || entryPointDefault;
+  const outDir = flags["o"] || flags["out"] || outDirDefault;
+
+  if (!version) {
+    printHelpText();
+    return 1;
+  }
+
+  await emptyDir(outDir);
+
+  await build({
+    entryPoints: [entryPoint],
+    outDir,
+    compilerOptions: { target: "ES2021" },
+    shims: {
+      // see JS docs for overview and more options
```

Comment:
I'm confused, which js docs? I think a link might help as from the above docs this doesn't seem to have an impact (no option to add more shims) on the script, or maybe it was copied and pasted, so we can delete it?